jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+|-------------|-----------------------------------------------------------------------------------------------|---------------|
+| description | Optional. Used to generate documentation for your orb.                                        | N/A           |
+| type        | Required. Currently "string", "boolean", "enum", and "steps" are supported                           | N/A           |
+| default     | The default value for the parameter. If not present, the parameter is implied to be required. | N/A           |
+
+### Parameter Types
+{:.no_toc}
+
+This section describes the types of parameters and their usage.
+
+#### String
+{:.no_toc}
+
+Basic string parameters are described below:
+
+```yaml
+commands:
+  copy-markdown:
+    parameters:
+      destination:
+        description: destination directory
+        type: string
+        default: docs
+    steps:
+      - cp *.md << destination >>
+```
+
+Strings should be quoted if they would otherwise represent another type (such as boolean or number) or if they contain characters that have special meaning in yaml. In all other instances, quotes are optional.
+
+#### Boolean
+{:.no_toc}
+
+Boolean parameters are useful for conditionals:
+
+```yaml
+commands:
+  list-files:
+    parameters:
+      all:
+        description: include all files
+        type: boolean
+        default: false
+    steps:
+      - ls <<# all >> -a <</ all >>
+```
+
+Boolean parameter evaluation is based on the [values specified in YAML 1.1][http://yaml.org/type/bool.html]:
+
+* true: `y` `yes` `true` `on`
+* false: `n` `no` `false` `off`
+
+Capitalized and uppercase versions of the above values are also valid.
+
+#### Steps
+{:.no_toc}
+
+Steps are used when you have a job or command that needs to mix predefined and user-defined steps. When passed in to a command or job invocation, the steps passed as parameters are always defined as an array, even if only one step is provided.
+
+```yaml
+commands:
+  run-tests:
+    parameters:
+      after-deps:
+        description: "Steps that will be executed after dependencies are installed, but before tests are run"
+        type: steps
+        default: []
+    steps:
+    - run: make deps
+    - steps: << parameters.after-deps >>
+    - run: make test
+```
+
+Steps passed as parameters are given as the value of a `steps` declaration under the job's `steps` declaration and are expanded and spliced into the array of existing steps. For example,
+
+```yaml
+jobs:
+  build:
+    machine: true
+    steps:
+    - run-tests:
+        after-deps:
+          - run: echo "The dependencies are installed"
+          - run: echo "And now I'm going to run the tests"
+```
+
+will become:
+
+```yaml
+steps:
+  - run: make deps
+  - run: echo "The dependencies are installed"
+  - run: echo "And now I'm going to run the tests"
+  - run: make test
+```
+
+#### Enum Parameter
+{:.no_toc}
+
+Use the `enum` parameter to declare the target operating system for a binary.
```

Comment:
This is describing the particular example below -- somehow that example ended up as the description, but it's not accurate. Something more like: "Use the `enum` parameter type when you want to enforce that the value must be one from a specific set of string values"

Reply:
added, thanks